src/api/Comments.js

```diff
@@ -5,7 +5,17 @@
  */
 
 import OffsetBasedAPI from './OffsetBasedAPI';
-import { PERMISSION_CAN_COMMENT, PERMISSION_CAN_DELETE, PERMISSION_CAN_EDIT } from '../constants';
+import {
+    PERMISSION_CAN_COMMENT,
+    PERMISSION_CAN_DELETE,
+    PERMISSION_CAN_EDIT,
+    ERROR_CODE_CREATE_COMMENT,
```

Comment:
Would be great to put these behind a namespace, like `ERROR_CODE.CREATE_COMMENT` or `PERMISSION.CAN_COMMENT`